### DISCUSSION TITLE
Feature Request: Dead Zone for Window Throw Trigger

### DISCUSSION THREAD

**Discussion by CVYY39** [rxhanson/RectanglePro-Community#31] [Ideas]
The window throw trigger is great, but it needs a dead zone so that you can more easily continue using that button for other things. I.e. if you assign it to the middle button right now, it becomes hard to use that button to close tabs in a browser or for other things. It would be better if it didn't engage until you had moved the mouse maybe 10 pixels in any direction.

**Comment by rxhanson** [comment]
Try adjusting the Min dist to appear:

![F2E3DC38-581F-4535-967D-B6A8A1798D0C](https://user-images.githubusercontent.com/13651296/156638233-b4245470-1819-408a-bded-955768837482.png)

**Comment by CVYY39** [comment]
Thank you, that fixed it. I had no idea there were options hiding behind those symbols - might need a UI tweak. I also think you might be better off defaulting to >0 min dist to appear setting.
